pkg/util/misc.go

```diff
@@ -588,6 +588,7 @@ func initInternalClient() {
 	}
 	internalHTTPSchema = "https"
 	internalHTTPClient = &http.Client{
+		Timeout:   5 * time.Minute,
```

Comment:
Do we need to add the timeout to the line586 too？

Reply:
PTAL